hera_pspec/uvpspec.py

```diff
@@ -1967,18 +1967,18 @@ def average_spectra(self, blpair_groups=None, time_avg=False,
             not specified is thrown out of the new averaged object.
 
         error_weights: string, optional
-            error_weights specify which kind of errors we use for weights 
-            during averaging power spectra. The weights are defined as 
-            $w_i = 1/ sigma_i^2$, where $sigma_i$ is taken from the relevant 
+            error_weights specify which kind of errors we use for weights
+            during averaging power spectra. The weights are defined as
+            $w_i = 1/ sigma_i^2$, where $sigma_i$ is taken from the relevant
             field of stats_array.
-            
-            If `error_weight` is set to `None`, which means we just use the 
+
+            If `error_weight` is set to `None`, which means we just use the
             integration time as weights. If `error_weights` is specified,
             then it also gets appended to `error_field` as a list.
             Default: None.
 
         inplace : bool, optional
-            If True, edit data in self, else make a copy and return. 
+            If True, edit data in self, else make a copy and return.
```

Comment:
same question here: is anything actually changed here?

Reply:
Nothing actually changed (my editor removed a space).